draft-ietf-quic-qpack.md

```diff
@@ -1144,7 +1179,193 @@ QPACK_DECODER_STREAM_ERROR (0x202):
+can be ineffectual if the attacker has a reliable way of causing values to be
+reinstalled. For example, a request to load an image in a web browser typically
+includes the Cookie header field (a potentially highly valued target for this
+sort of attack), and web sites can easily force an image to be loaded, thereby
+refreshing the entry in the dynamic table.
+
+This response might be made inversely proportional to the length of the header
+field value. Marking a header field as not using the dynamic table any more
+might occur for shorter values more quickly or with higher probability than for
+longer values.
+
+## Never Indexed Literals
+
+Implementations can also choose to protect sensitive header fields by not
+compressing them and instead encoding their value as literals.
+
+Refusing to insert a header field into the dynamic table is only
+effective if doing so is avoided on all hops. The never indexed literal bit (see
+{{literal-name-reference}}) can be used to signal to intermediaries that a
+particular value was intentionally sent as a literal.
+
+An intermediary MUST NOT re-encode a value that uses a literal representation
+with the 'N' bit set with another representation that would index it. If QPACK
+is used for re-encoding, a literal representation with the 'N' bit set MUST be
+used.  If HPACK (see Section 6.2.3 of [RFC7541]) is used for re-encoding, the
+never indexed literal representation MUST be used.
+
+The choice to use a never indexed literal representation for a header field
+depends on several factors. Since QPACK doesn't protect against guessing an
+entire header field value, short or low-entropy values are more readily
+recovered by an adversary. Therefore, an encoder might choose not to index
+values with low entropy.
+
+An encoder might also choose not to index values for header fields that are
+considered to be highly valuable or sensitive to recovery, such as the Cookie or
+Authorization header fields.
+
+On the contrary, an encoder might prefer indexing values for header fields that
+have little or no value if they were exposed. For instance, a User-Agent header
+field does not commonly vary between requests and is sent to any server. In that
+case, confirmation that a particular User-Agent value has been used provides
+little value.
+
+Note that these criteria for deciding to use a never indexed literal
+representation will evolve over time as new attacks are discovered.
+
+##. Static Huffman Encoding
+
+There is no currently known attack against a static Huffman encoding. A study
+has shown that using a static Huffman encoding table created an information
+leakage, however this same study concluded that an attacker could not take
+advantage of this information leakage to recover any meaningful amount of
+information (see [PETAL]).
+
+## Memory Consumption
+
+An attacker can try to cause an endpoint to exhaust its memory. QPACK is
+designed to limit both the peak and state amounts of memory allocated by an
+endpoint.
```

Comment:
```suggestion
designed to limit both the peak and stable amounts of memory allocated by an
endpoint.
```

Reply:
This error is in the HPACK RFC.  Is it worth an errata?

Reply:
I'll let you manage that :)  You can be the author of an RFC erratum.  The glory!